doc/ref/modules/all/salt.modules.flatpak.rst

```diff
@@ -0,0 +1,6 @@
+=================
+salt.modules.flatpak
+=================
```

Comment:
Sorry, I missed this one... The `=` needs to be exactly as long as the string inside, ie:
```
====================
salt.modules.flatpak
====================
```